modules/storage-aws-s3/s3/s3.go

```diff
@@ -93,6 +97,14 @@ func (s *s3) StoreSnapshot(ctx context.Context, snapshot *snapshots.Snapshot) er
 			return snapshots.NewErrInternal(
 				errors.Wrapf(err, "put file '%s'", objectName))
 		}
+
+		// Get size of file
+		fileInfo, err := os.Stat(filePath)
+		if err != nil {
+			return errors.Errorf("Unable to get size of file %v", filePath)
+		}
+		monitoring.GetMetrics().SnapshotStoreDataTransferred.WithLabelValues("s3", snapshot.ClassName).Add(float64(fileInfo.Size()))
```

Comment:
this metric is being pushed for every file, I see that you're creating a metric with `s3` and `className` label and assigning it a size, we don't take into account `file's name`? and this is a cumulative metric that holds the sum of all file sizes?

Reply:
oh, so it's a cumulative metric yes? calculated separately for `store` and `restore` operations, now I see it

Reply:
yes.  I can add filenames if you feel they are important, but the guideline from grafana is to avoid creating a lot of labels